ext/oj/dump.c

```diff
@@ -576,7 +576,7 @@ void oj_dump_xml_time(VALUE obj, Out out) {
             format[32] = '0' + out->opts->sec_prec;
             len -= 9 - out->opts->sec_prec;
         }
-        sprintf(buf,
+        len = sprintf(buf,
```

Comment:
Same on this change. Best to remove the earlier calculation of Len since it is over-ridden by this line.

Reply:
Thanks. I removed redundant codes.